README.md

```diff
@@ -1,39 +1,45 @@
 # fablicop
 
 fablicop is a RuboCop configration gem.
+It assumes your project is using Ruby on Rails, so some Cops prefixed with `Rails` are enabled out of the box.
+In other words, it's not appropriate to use it with non-Rails projects.
 
-## Usage
+## Installation
+
+You can install fablicop with this command:
+
+```console
+gem install fablicop
+```
 
-Setup .rubocop.yml
+Or, run `bundle install` after adding this line to your application's Gemfile:
 
-```sh
-bundle exec fablicop init
+```ruby
+gem 'fablicop', require: false
```

Comment:
I deliberately removed `group :development do ... end` because it depends on a project that the gem should be surrounded with `group`.